en/02_Developer_Guides/15_Customising_the_Admin_Interface/01_ModelAdmin.md

```diff
@@ -146,6 +144,12 @@ class MyAdmin extends ModelAdmin
 
 It is trivial to get links to the edit form for managed records.
 
+> [!NOTE]
+> The [getLinkForModelClass()](api:SilverStripe\Admin\ModelAdmin::getLinkForModelClass()) method returns a link
+> for the first tab defined for that class. If you have multiple tabs for a given class (as in the example above)
+> it is better to use [getLinkForModelTab()](api:SilverStripe\Admin\ModelAdmin::getLinkForModelTab()) which will
+> give you a link for the specific tab you pass in.
```

Comment:
Moved this from below to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.